### PR TITLE
check-kmod-load-unload: fix all shellcheck warnings

### PR DESCRIPTION
**Pull request by marc-hb**
No functional change for now, not even a minor bug fix.

Signed-off-by: Marc Herbert <marc.herbert@intel.com>

### CODE REVIEW

File: test-case/check-kmod-load-unload.sh [hunk 1]
```diff
@@ -33,12 +34,9 @@ func_opt_parse_option "$@"
 func_lib_setup_kernel_last_line
 
 loop_cnt=${OPT_VALUE_lst['l']}
-usb_audio_module="snd_usb_audio"
-kern_log="/var/log/kern.log"
-keyword_info=""
 
 PATH="${PATH%%:*}/kmod:$PATH"
-func_lib_check_sudo
+func_lib_check_sudo 'unloading modules'
```
Comment (marc-hb) [inline test-case/check-kmod-load-unload.sh:39]: This is required to avoid a warning once #369 is merged.

**Comment by marc-hb** [comment]
The only device-test failure in https://sof-ci.01.org/softestpr/PR368/build163/devicetest is ICL issue #365

**Review by aiChaoSONG — APPROVED** [review]
LGTM

**Comment by marc-hb** [comment]
Thanks @aiChaoSONG . No functional change, merging.

**Pull request merged**